doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3682,7 +3682,7 @@ The issues in this section are related to the TF-M implementation in the |NCS|.
 NCSDK-22818: TF-M does not build with :kconfig:option:`CONFIG_TFM_PROFILE_TYPE_MINIMAL` disabled and :kconfig:option:`CONFIG_TFM_LOG_LEVEL_SILENCE` enabled
   This fails with the error: message "undefined reference to 'end'" caused by printf being included in the build.
 
-.. rst-class:: v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0
+.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0
```

Comment:
double-checked. This feedback looks correct to me.

I proposed removing NCSDK-22629 from affected versions.
Joakim has found that NCSDK-22629 was the wrong jira issue and has a
PR for fixing this.

My proposal is valid because the NCS text says that nrfjprog
10.22.x is affected, but 10.23.x is the latest version and I
assume that is what is installed and supported.

Georgios proposes adding it to the affected version but I don't think
that it is correct.